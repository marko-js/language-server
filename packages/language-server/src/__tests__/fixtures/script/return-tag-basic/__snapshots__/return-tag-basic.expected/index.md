## Diagnostics
### Ln 2, Col 2
```marko
  1 | <let/value = 1/>
> 2 | <button onClick() {
    |  ^^^^^^ Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"
  3 |   value++;
  4 | }/>
  5 |
```

