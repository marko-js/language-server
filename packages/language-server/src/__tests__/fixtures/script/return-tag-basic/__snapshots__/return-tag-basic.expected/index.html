<button data-marko-node-id="1" onClick></button>